host/cli/signal.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/pkg/cluster"
+)
+
+func init() {
+	Register("signal", runSignal, `
+usage: flynn-host signal ID [SIGNAL]
```

Comment:
The code requires a valid `SIGNAL` argument, so it shouldn't be wrapped in `[ ]`